### PR TITLE
Create docs for ViewPresenter in fundamentals

### PR DESCRIPTION
**Pull request by nmilcoff**
I've taken some ideas from: 
- http://gregshackles.com/presenters-in-mvvmcross-a-primer/

Hope @gshackles is ok with that :)

### CODE REVIEW

File: docs/_documentation/fundamentals/view-presenters.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Let's take a look now at the methods of that interface:
+
+- `Show(MvxViewModelRequest request)`: This method is in charge of deciding how a view should be presented and then show it. It takes a ViewModel request as parameter, which contains all the the information passed in `ShowViewModel`.
+- `AddPresentationHintHandler<THint>(Func<THint, bool> action) where THint : MvxPresentationHint`: You can use this method to register your custom PresentationHints and their respective handlers (we will explore the PresentationHint concept a bit later).
+- `ChangePresentation(MvxPresentationHint hint)`: This method is the one that gets called whenever a PresentationHint is requested. It is responsible for handling the requested change.
+- `Close(IMvxViewModel toClose)`: As you can imagine, this method is used to handle the close request of a ViewModel. It takes the ViewModel instance to be closed as parameter.
+
+
+##Showing ViewModels
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/view-presenters.md]: Needs spacing between ## and "Showing ViewModels"

File: docs/_documentation/fundamentals/view-presenters.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    public partial class MyView : MvxViewController<MyViewModel>
+    {
+        // your UI code
+    }
+}
+```
+
+In this example the attribute is telling the Presenter that whenever a ViewModel request for `MyViewModel` arrives, it should display the View as a root and wrap it into a NavigationController (it initiates a navigation stack).
+
+##Requesting presentation changes
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/view-presenters.md]: Same here

**Comment by nmilcoff** [comment]
@MarcBruins thank you! It's done :)

**Comment by nmilcoff** [comment]
Oops! I should have made the PR to master branch instead of develop, right? Should I open a new one to master?

**Comment by martijn00** [comment]
No worries, i'll merge this stuff into master together with everything else.

**Pull request merged**